bin/bnf_to_cnf/poetry.lock

```diff
@@ -47,7 +47,7 @@ description = "A utility for ensuring Google-style docstrings stay up to date wi
 name = "darglint"
 optional = false
 python-versions = ">=3.5"
-version = "1.1.0"
+version = "2.0.0"
```

Comment:
The hash further below in this file of course needs to be changed. Maybe we'll just regenerate the whole file using Poetry. I did try it out, and the contents changed quite a bit – including the sections generated in the file.

Reply:
I'll deal with this later separately.